docs/how-to/fuzzing-dotnet-with-libfuzzer.md

```diff
@@ -0,0 +1,222 @@
+# Fuzzing .Net using libfuzzer-dotnet
+
+Fuzzing .Net using libFuzzer makes use of three Open Source capabilities.
```

Comment:
Is this only for dotnet core library ? if that is the case we should probably say it this doc